### DISCUSSION TITLE
Getting Root node is not accessible Error while applying

### DISCUSSION THREAD

**Discussion by vishnuvt999** [Netcentric/accesscontroltool#706] [General]
Hi Team,

I have installed NetCentric ACL Tool in my local and Getting Root node is not accessible Error while applying .

Version :- 6.5.20
Netcentric Tool:- AC Tool v3.0.9
Added admin privilege to **actool-service** user also 
![ACL-ERROR](https://github.com/Netcentric/accesscontroltool/assets/168809927/4575ee32-82b3-4bcb-88d8-def78e8e136c)

03.05.2024 11:32:38.408 *ERROR* [[0:0:0:0:0:0:0:1] [1714716158359] GET /mnt/overlay/netcentric/actool/content/overview.html/actool HTTP/1.1] biz.netcentric.cq.tools.actool.history.impl.AcHistoryServiceImpl Could not get history items: javax.jcr.AccessDeniedException: Root node is not accessible.
javax.jcr.AccessDeniedException: Root node is not accessible.
	at org.apache.jackrabbit.oak.jcr.session.SessionImpl$5.perform(SessionImpl.java:324) [org.apache.jackrabbit.oak-jcr:1.22.18]
	at org.apache.jackrabbit.oak.jcr.session.SessionImpl$5.perform(SessionImpl.java:318) [org.apache.jackrabbit.oak-jcr:1.22.18]
	at org.apache.jackrabbit.oak.jcr.delegate.SessionDelegate.perform(SessionDelegate.java:207) [org.apache.jackrabbit.oak-jcr:1.22.18]
	at org.apache.jackrabbit.oak.jcr.session.SessionImpl.getRootNode(SessionImpl.java:318) [org.apache.jackrabbit.oak-jcr:1
	
	Please advice

**Comment by kwin** [comment]
This sounds like lack of permissions of the underlying service user. Please try to install the service user permissions as they ship with ACTool (https://github.com/Netcentric/accesscontroltool/tree/develop/accesscontroltool-content-package/src/main/jcr_root).

**Reply by vishnuvt999** [reply]
Thanks @kwin  for the reply 

Can you please share me the .zip file location

**Comment by kwin** [comment]
https://repo1.maven.org/maven2/biz/netcentric/cq/tools/accesscontroltool/accesscontroltool-content-package/3.0.9/accesscontroltool-content-package-3.0.9.zip but it is recommended to use the all package (https://repo1.maven.org/maven2/biz/netcentric/cq/tools/accesscontroltool/accesscontroltool-package/3.0.9/accesscontroltool-package-3.0.9.zip) which includes the former. Compare with https://github.com/Netcentric/accesscontroltool/blob/develop/docs/Installation.md#installation-aem-on-premise

**Reply by vishnuvt999** [reply]
Thanks @kwin  It's started working .

Applied a sample config also 

Will this ACL tool work for exporting Current permission as Backup file or Excel file ?

**Reply by vishnuvt999** [reply]
@kwin Thanks i have downloaded the dump 


Changed some permissions and while trying to submit   getting SUCCESS (0 authorizables/0 ACLs changed). 

Please advice 

![ACL-ERROR1](https://github.com/Netcentric/accesscontroltool/assets/168809927/6ff1e8ed-8fd6-477a-85d0-688ecc3d3c7c)
